wallet/src/main/scala/org/bitcoins/wallet/models/AddressTagTable.scala

```diff
@@ -0,0 +1,47 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.protocol.BitcoinAddress
+import org.bitcoins.core.wallet.utxo.AddressTag
+import slick.jdbc.SQLiteProfile.api._
+import slick.lifted.{PrimaryKey, ProvenShape}
+
+abstract class AddressTagDb[AddressTagType <: AddressTag] {
+  def address: BitcoinAddress
+
+  def tag: AddressTagType
+}
+
+abstract class AddressTagTable[
+    AddressTagType <: AddressTag,
+    AddressTagDbType <: AddressTagDb[AddressTagType]](t: Tag, tableName: String)
+    extends Table[AddressTagDbType](t, tableName) {
```

Comment:
Any reason for not using `TableAutoInc`

Reply:
No, we don't on any tables currently #1185